prototypes/buildings/arthurian-pen.lua

```diff
@@ -542,43 +68,50 @@ RECIPE {
     }
 }:add_unlock("land-animals-mk04")
 
+local FULL_CRAFTING_SPEED = 1 -- crafting speed when full of mk01 modules
+local MODULE_SLOTS = 8
+local NAME = "arthurian-pen-mk0"
+local SUBGROUP = "py-alienlife-farm-buildings-mk0"
+local ENERGY_USAGE = {"200kW","300kW", "500kW", "700kW"}
```

Comment:
I think given that all caps is used for global functions like `ITEM`, lower snake case works better here.